File: backend/src/main/java/com/livana/backend/proof/dto/ProofDto.java
package com.livana.backend.proof.dto;

import java.time.OffsetDateTime;

public record ProofDto(
    int proofId,
    String ipfsCid,
    long amount,
    boolean released,
    OffsetDateTime submittedAt,
    OffsetDateTime releasedAt
) {}

File: android/app/src/main/java/com/livana/app/core/network/dto/ProofDto.kt
package com.livana.app.core.network.dto

import kotlinx.serialization.Serializable

@Serializable
data class ProofDto(
    val proofId: Int,
    val ipfsCid: String,
    val amount: Long,
    val released: Boolean,
    val submittedAt: String,
    val releasedAt: String?,
)
